projects/data-grid-demo/src/app/app.component.ts

```diff
@@ -100,12 +100,9 @@ export class AppComponent implements OnInit {
       field: 'age',
       component: NumberComponent,
       visible: true,
-      sortAvailable: true,
-      filterAvailable: true,
-      filter: {
```

Comment:
Is this removed by mistake?

Reply:
No. It's removed in order to show small column without sort and filter buttons.